src/utlis/utlis.js

```diff
@@ -0,0 +1,3 @@
+export const calculateDiscount = (num, num2) => {
+    return Math.floor(100 - (num / num2) * 100);
+}
```

Comment:
Great thing that you used utils to keep that function. It's pretty nifty to have it stored outside of SliderItem scope.